options/secrets_test.go

```diff
@@ -0,0 +1,150 @@
+package options_test
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/coder/envbuilder/options"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetBuildSecrets(t *testing.T) {
```

Comment:
Suggestion: In coder/coder we usually add comments like this:
```go
//nolint:tparallel // This test modifies the environment.
```
It seems we don't have this linter enabled here, but it's good practice to add a note about why we're not doing a parallel test.

Reply:
I'd prefer to enable the linter before adding the comment, because that way we can confirm that it works. I've created a separate issue for updating the linter config though, because if it finds more than expected I don't want that to extend this PR. I've created #399 to deal with linting for envbuilder, including this comment and the other one in this PR.